frontend/src/relay/auth.tsx

```diff
@@ -58,7 +59,11 @@ export const ExternalLink: React.FC<ExternalLinkProps> = ({
             link: () => <Link className={className} to={target.toString()} target="_blank">
                 {children}
             </Link>,
-            button: () => <LinkButton className={className} to={target.toString()} target="_blank">
+            button: () => <LinkButton
+                className={className}
+                to={target.toString()}
+                target="_blank"
+            >
```

Comment:
Nothing changed here, right? Is this change intended?

Reply:
Oh right, I used a different indentation first where line 61 would exceed 100 chars. No longer the case though.